grunt-configs/webfontjson.js

```diff
@@ -584,6 +586,6 @@ module.exports = function(grunt, options) {
                     }
                 ]
             }
-        }
+        },
     };
 };
```

Comment:
yeah i removed all the fonts form this PR because it was crashing people's browsers. that meant the webfontjson task failed the build because it couldn't find the file. then i left that BLOODY COMMA there :) will remove it